client/app/telecom/telephony/alias/special/fees/telecom-telephony-alias-special-fees.html

```diff
@@ -0,0 +1,124 @@
+<section class="telephony-group-alias-special-fees">
+    <header>
+        <oui-back-button data-ui-sref="telecom.telephony.alias.special"
+                         heading="{{:: 'telephony_alias_special_fees_title' | translate }}">
+        </oui-back-button>
+    </header>
+
+    <div class="telecom-section-content">
+
+        <toast-message></toast-message>
+
+        <div class="row">
+            <div class="col-md-6 text-center"
+                 data-ng-if="$ctrl.isLoading">
+                <oui-spinner></oui-spinner>
+            </div>
+
+            <div class="col-md-6 widget-presentation"
+                 data-ng-if="!$ctrl.isLoading">
+                <table class="oui-datagrid">
```

Comment:
In order to migrate to `<oui-datagrid>` we developed `ovh-ui-kit-bs` dependency (Bootstrap theme) that give us the style of the `<oui-datagrid>`. Why not using a regular `<table class="table">` instead?

Reply:
In fact, I restyled the table (who was table-responsive) to oui-datagrid classes, in order to make it more acceptable visually.